CONTRIBUTION.md

```diff
@@ -16,12 +16,26 @@ Please note we have a code of conduct, please follow it in all your interactions
 
 ### Setup Project for Development and Testing
 
-1. Follow the package setup instructions on the main README till the [`Clone Project Repository`](README.m#131-clone-project-repository) step.
-2. If you are a `schematic` package contributor you don't need to create a `venv` virtual environment, `poetry` will create a virtual environment by default, which you can use.
-3. Running the following command reads the [`pyproject.toml`](https://github.com/Sage-Bionetworks/schematic/blob/develop/pyproject.toml) file from the current project, resolves the dependencies and installs them: `poetry install`
-4. Obtain credentials file(s) by following same instructions as in the [`1.3.4. Obtain Credentials File(s)`](README.md#134-obtain-credentials-files) section.
-5. Fill in configuration file(s) in the same way as specified in the [`1.3.5. Fill in Configuration File(s)`](README.md#135-fill-in-configuration-files).
-6. To run any of the CLI utilities shown in the [`1.3.6. Command Line Interface`](README.md#136-command-line-interface), prefix the commands with `poetry run`.
+All the sections referenced here are from the [documentation](https://sage-schematic.readthedocs.io/en/develop/index.html) on ReadTheDocs.
+
+1. Make sure you have all the packages and tools required as specified in the [`Installation Requirements and Pre-requisites`](https://sage-schematic.readthedocs.io/en/develop/README.html#installation-requirements-and-pre-requisites) section.
+2. Clone the `schematic` package repository from GitHub onto your local machine by running the following command: `git clone --single-branch --branch develop https://github.com/Sage-Bionetworks/schematic.git`
```

Comment:
Why `--single-branch`? And why `--branch develop`? Isn't `develop` our default branch anyways? 

Also, I wonder if it's better to use the SSH link over HTTPS.

Reply:
Good catch. Yup, we needed to use this when `main` was still our default branch, we don't need it now.

Apparently, GitHub now recommends HTTPS over SSH: https://stackoverflow.com/questions/11041729/why-does-github-recommend-https-over-ssh